Formula/tideways-daemon.rb

```diff
@@ -13,14 +13,14 @@ class TidewaysDaemon < Formula
     sha256 checksum[arch]
 
     def install
-      bin.install 'tideways-daemon'
+    bin.install 'tideways-daemon'
```

Comment:
You removed the function indentation here while keeping it in the other files. Issue with auto formatting of the template due to the missing quote?

Reply:
Probably an issue with the automated indentation detection while copy-and-pasting into a file with different settings (2 vs 4).